data/data/aws/master/main.tf

```diff
@@ -96,7 +96,7 @@ resource "aws_instance" "master" {
   tags = "${merge(map(
       "Name", "${var.cluster_name}-master-${count.index}",
       "kubernetes.io/cluster/${var.cluster_name}", "owned",
-      "tectonicClusterID", "${var.cluster_id}",
+      "openshiftClusterID", "${var.cluster_id}",
```

Comment:
whoa, boy ;).  This one is quasi-public, so I wouldn't be surprised if we break some third-party tooling with this change.  Still, I think dropping `tectonic` makes sense to get consistent branding.  And `openshift` is probably appropriate here, although I'd be open to `"kubernetes.io/cluster-id", ${var.cluster_id}` if we could drum up an appetite for a cluster UUID tag at the vanilla Kubernetes level.

Reply:
I have removed the change of `tectonicClusterID` from this PR. I will do it in a subsequent PR.